examples/with-postgresql/components/button-link/index.tsx

```diff
@@ -0,0 +1,27 @@
+import Link from 'next/link'
+import cn from 'clsx'
```

Comment:
Isn't it better to avoid any unnecessary additional libs for examples?

Reply:
yep